### ISSUE TITLE
[Documentation]: Doxyfile is outdated

### ISSUE THREAD

**Issue by e-kwsm** [ROCm/ROCmValidationSuite#1095]
### Description of errors

- master 8fa7c36c237e9dddf0886b4130a1a86e79a0cb6d
- doxygen 1.16.1

During make, the following warnings are emitted:

```
[  4%] Generating RVS User Guide
warning: Tag 'TCL_SUBST' at line 247 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'COLS_IN_ALPHA_INDEX' at line 1098 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'HTML_TIMESTAMP' at line 1234 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'FORMULA_TRANSPARENT' at line 1515 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'LATEX_SOURCE_CODE' at line 1803 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'LATEX_TIMESTAMP' at line 1819 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'RTF_SOURCE_CODE' at line 1885 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOCBOOK_PROGRAMLISTING' at line 1983 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'PERL_PATH' at line 2163 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'CLASS_DIAGRAMS' at line 2176 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'MSCGEN_PATH' at line 2185 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOT_FONTNAME' at line 2227 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOT_FONTSIZE' at line 2234 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: Tag 'DOT_TRANSPARENT' at line 2455 of file 'DoxyGuide' has become obsolete.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u"
warning: tag INPUT: input source 'doc/ugsrc/' does not exist
warning: source 'doc/ugsrc' is not a readable file or directory... skipping.
warning: No files to be processed, please check your settings, in particular INPUT, FILE_PATTERNS, and RECURSIVE
warning: Specified markdown mainpage 'doc/ugsrc/ug1main.md' does not exist
```

doc/ugsrc seems to be removed at some point.

**Comment by jkottiku** [comment]
Will check and correct this.
